### ISSUE TITLE
2nd forwarded port does not appear

### ISSUE THREAD

**Issue by hpirila** [ThomDietrich/home-assistant-addons#21]
Hi,

I upgraded to 1.3.1 and after that the other port forwarding does not appear any more.

Conviguration before:
```
hostname: myserver.myhost.com
ssh_port: 22
username: autossh
remote_forwarding:
  - 127.0.0.1:8611:172.30.32.1:8123
  - 127.0.0.1:9611:172.30.32.1:22
other_ssh_options: "-v -N -o UserKnownHostsFile=/dev/null"
force_keygen: false
```
Configuration after upgrade:
```
hostname: myserver.myhost.com
ssh_port: 22
username: autossh
remote_ip_address: 127.0.0.1
remote_port: 8611
other_ssh_options: "-v -N -o UserKnownHostsFile=/dev/null"
force_keygen: false
remote_forwarding:
  - 127.0.0.1:9611:172.30.32.1:22
 
```

Now the ssh port 9611 does not open any more and I can't ssh back to HA.

In the log I don't see 9611 attempted at all.

```
[10:11:44] INFO: Preparations done.

[10:11:44] INFO: Executing command: /usr/bin/autossh -M 0 -o ServerAliveInterval=30 -o ServerAliveCountMax=3 -o StrictHostKeyChecking=no -o ExitOnForwardFailure=yes -p 22 -t -t -i /data/ssh_keys/autossh_rsa_key autossh@myserver.myhost.com -R 127.0.0.1:8611:172.30.32.1:8123 -v -N -o UserKnownHostsFile=/dev/null
```

**Comment by hpirila** [comment]
I restored back version 1.2.1. This works and the log shows like this:

```
[10:28:22] INFO: Preparations done.

[10:28:22] INFO: Executing command: /usr/bin/autossh  -M 0 -o ServerAliveInterval=30 -o ServerAliveCountMax=3 -o StrictHostKeyChecking=no -o ExitOnForwardFailure=yes -p 22 -t -t -i /data/ssh_keys/autossh_rsa_key autossh@myserver.myhost.com -R 127.0.0.1:8611:172.30.32.1:8123 -R 127.0.0.1:9611:172.30.32.1:22 -v -N -o UserKnownHostsFile=/dev/null
```

**Comment by ThomDietrich** [comment]
Thanks for the report!! I've just released v1.3.2 which fixes this bug.
Beyond your issue this bug also broke the backwards compatibility. Now users should be able to update and reconnect with their old config, as intended.

Cheers!
